.chalice/config_template.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "2.0",
+    "app_name": "fusillade",
+    "stages": {
+    },
+    "lambda_timeout": 10,
+    "lambda_memory_size": 1536,
+    "tags": {
+        "managedBy": "chalice",
+        "Name": "fusillade"
```

Comment:
does this tag need to be changed to service like the config.json file?

Reply:
it sure does